code/src/index.css

```diff
@@ -7,7 +13,100 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+.Outline{
+  background-color: rgb(24, 24, 22);
+}
+
+.Album-Wrapper{                 /*background color behind the artists*/
+  background-color: #181818;
+  width: 100vw;
+  padding-top: 40px;
+  margin-right: 40px;
+  border: solid 30px;
+
+}
+
+h1 {
+  color: white;
+  margin-left: 10px;
+}
+.Box{
+  color:  white;
+  width: 300px;
+  height: 300px;
+}
+
+.Album{
```

Comment:
the design specs said it should only be 4 covers per row on desktop and now it fills out the entire screen with as many as has space. So on mine for example it has 6 covers in a row on my desktop. I suggest you add a grid display styling for the covers so you can specifiy how many per row for each media size :)